java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -76,6 +76,21 @@ public interface SetBaseCommands {
      */
     CompletableFuture<Long> scard(String key);
 
+    /**
+     * Gets the intersection of all the given sets.
+     *
+     * @see <a href="https://redis.io/commands/sinter/">redis.io</a> for details.
+     * @param keys The keys of the sets.
+     * @return A <code>Set</code> of members which are present in all given sets.<br>
+     *     Missing or empty input sets cause an empty set response.
+     * @example
+     *     <pre>{@code
+     * String[] values = client.sinter(new String[] {"list1", "list2"}).get();
+     * assert values.length > 1; // Indicates that these sets have at least one common element
```

Comment:
what about checking that `member` is in the resulted set?

Reply:
That case isn't verbose, because `sinter` returns empty result when there are no common elements in the sets too.